core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycleSpec.scala

```diff
@@ -392,6 +397,60 @@ class BpmBillingProjectLifecycleSpec extends AnyFlatSpec {
     }
   }
 
+  it should "return the original error if resource deletion also errors" in {
```

Comment:
Test case that the correct exception is passed through, even if cleanup also throws an exception.